scripts/build/llvm-src.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -x
+set -ev
+
+DIR=$(dirname "$(readlink -f "$0" || stat -f "$0")")
+source "${DIR}/common-defaults.sh"
+
+if [[ "${LLVM_VERSION_SHORT}" == "" ]]; then
+  echo "LLVM_VERSION_SHORT not set"
+  exit 1
+fi
+
+if [[ "${BASE}" == "" ]]; then
+  echo "BASE not set"
+  exit 1
+fi
+if [[ "${PACKAGED}x" == "1x" ]]; then
+  echo "LLVM: packaged version"
+  exit 0
+fi
+LLVM_BASE="${BASE}/llvm-${LLVM_VERSION_SHORT}"
+
+# Checkout LLVM code
+svn co http://llvm.org/svn/llvm-project/llvm/branches/release_${LLVM_VERSION_SHORT} "${LLVM_BASE}"
```

Comment:
Why use SVN, and not the zipped sources? I think `wget` and `tar` should be significantly faster than `svn co`

Reply:
I was doing this based on the install instructions of LLVM.
I agree it could be faster. It's not painful slow either.